### PR TITLE
chore: add placeholder user_id

### PR DESCRIPTION
**Pull request by ashwins93**

### CODE REVIEW

File: src/pages/PreviewScreen.jsx [hunk 1]
```diff
@@ -3,13 +3,26 @@ import { useHistory, useParams, useLocation } from "react-router-dom";
...
 const PreviewScreen = () => {
   const history = useHistory();
   const context = useContext(AppContext);
   const { loginInfo, setLoginInfo, setMaxTileCount, tokenEndpoint } = context;
   const { roomId: urlRoomId, role: userRole } = useParams();
   const location = useLocation();
+  const [token, setToken] = useState("");
+
+  useEffect(() => {
+    getToken(tokenEndpoint, loginInfo.env, "a", userRole, urlRoomId).then(
```
Comment (triptu) [inline src/pages/PreviewScreen.jsx:20]: can you add a comment here why "a" is necessary.

Comment (ashwins93) [reply src/pages/PreviewScreen.jsx:20]: This was causing the token generation service to return `null` for the token. Sending any non-empty string yields the token.

Comment (triptu) [reply src/pages/PreviewScreen.jsx:20]: and maybe we should change "a" to sth like "dummy" too, to convey the signficance.

Comment (triptu) [reply src/pages/PreviewScreen.jsx:20]: yes I know of the reason. It's more for future readers of the codebase.

Comment (triptu) [reply src/pages/PreviewScreen.jsx:20]: code answers what and comments answer why.

**Review by triptu — APPROVED** [review]

**Pull request merged**